packages/glsurf/glsurf-3.3/opam

```diff
@@ -0,0 +1,22 @@
+opam-version: "1.2"
+name: "glsurf"
+version: "3.3"
+maintainer: "Christophe Raffalli <raffalli@univ-savoie.fr>"
+bug-reports:  "mailto:raffalli@univ-savoie.fr"
+authors:
+  [ "Christophe Raffalli <raffalli@univ-savoie.fr>" ]
+homepage: "http://lama.univ-savoie.fr/~raffalli/glsurf.html"
+license: "GPL-3.0"
+dev-repo: "darcs://lama.univ-savoie.fr/~raffalli/glsurf/repos"
+build: [make]
+install: [make "install" "BINDIR=%{bin}%"]
+remove: [rm %{bin}%/glsurf]
```

Comment:
On 16-05-30 06:27:25, Fabrice Le Fessant wrote:
> In packages/glsurf/glsurf-3.3/opam:
>
> > @@ -0,0 +1,22 @@
> > +opam-version: "1.2"
> > +name: "glsurf"
> > +version: "3.3"
> > +maintainer: "Christophe Raffalli <raffalli@univ-savoie.fr>"
> > +bug-reports:  "mailto:raffalli@univ-savoie.fr"
> > +authors:
> > +  [ "Christophe Raffalli <raffalli@univ-savoie.fr>" ]
> > +homepage: "http://lama.univ-savoie.fr/~raffalli/glsurf.html"
> > +license: "GPL-3.0"
> > +dev-repo: "darcs://lama.univ-savoie.fr/~raffalli/glsurf/repos"
> > +build: [make]
> > +install: [make "install" "BINDIR=%{bin}%"]
> > +remove: [rm %{bin}%/glsurf]
>
> should be "%{bin}%/glsurf", no ?

And opam pin add would have told me before pull request ... sorry ...

> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.*
>